Products/ZenUI3/browser/resources/js/zenoss/Action.js

```diff
@@ -66,7 +59,6 @@
     });
 
     function setDisabled(disable) {
-        this.externallyDisabled = disable;
         this.callParent([this.checkDisabled()]);
     }
```

Comment:
Yes, if `disable` is `true` then a particular administered object will be disabled for a user, otherwise, it will be enabled. However, `externallyDisabled` is not used for this purpose, moreover, it is unused outside the Action.js. The issue was previously fixed [here](https://github.com/zenoss/zenoss-prodbin/pull/3902).